nautobot/extras/templatetags/job_buttons.py

```diff
@@ -108,16 +108,17 @@ def _render_job_button_for_obj(job_button, obj, context, content_type):
     # Disable buttons if the user doesn't have permission to run the underlying Job.
     has_run_perm = Job.objects.check_perms(context["user"], instance=job_button.job, action="run")
     try:
-        _task_queue = job_button.job.task_queues[0]
+        job_queues = list(job_button.job.job_queues.all().values_list("name", flat=True))
+        _job_queue = job_queues[0]
```

Comment:
Behavior change here if we no longer have a user-configurable ordering of `job.job_queues`. Document please!